scripts/test-react.js

```diff
@@ -142,6 +142,10 @@ function runTestSuite(outputJsx) {
         await runTest(directory, "simple-5.js");
       });
 
+      it("Simple 6", async () => {
```

Comment:
Should this really be a React test? Seems like a more generic feature that could maybe be in the main test suite.

I don't mean the existing test to be deleted, but maybe a new one in `additional-functions` could do?

Reply:
I've added a pure test in addition to the React test